extensions/kusto/src/kustoServer.ts

```diff
@@ -35,7 +45,7 @@ export class KustoServer {
 			const installationComplete = Date.now();
 			let serverOptions = generateServerOptions(context.extensionContext.logUri.fsPath, path);
 			let clientOptions = getClientOptions(context);
-			this.client = new SqlOpsDataClient(Constants.serviceName, serverOptions, clientOptions); // TodoKusto: Update constant
+			this.client = new SqlOpsDataClient('kusto', Constants.serviceName, serverOptions, clientOptions); // TodoKusto: Update constant
```

Comment:
Could you clarify this TODO more? (and even better file an issue for it if it's something that should actually be done)